hazelcast-client-new/src/test/java/com/hazelcast/client/ClientRegressionWithMockNetworkTest.java

```diff
@@ -627,9 +627,60 @@ public void entryAdded(EntryEvent<Object, Object> event) {
             map1.put(i, i);
         }
 
-        assertOpenEventually("dadas", latch);
+        assertOpenEventually(latch);
```

Comment:
ok, in this case the instance.shutdown() on the rest of the tests are redundant?